auth/server.go

```diff
@@ -0,0 +1,65 @@
+package auth
+
+import (
+	"encoding/json"
+	"net/http"
+
+	utils "github.com/fabric8-analytics/cli-tools/utils"
+)
+
+// RequestServerType is a argtype of RequestServer func
+type RequestServerType struct {
+	SynkToken       string
+	UserID          string
+	Host            string
+	ThreeScaleToken string
+	Debug           bool
+}
+
+// RequestServer is auth request to CRDA server
+func RequestServer(requestParams RequestServerType) string {
+	var reqPayload utils.UserRequest
+	var resData utils.UserResponse
+	requestData := utils.HTTPRequestType{
+		ReqData:         reqPayload,
+		Method:          http.MethodPost,
```

Comment:
I dont think method and endpoint need to be available here. This is just about collecting user configured information. The rest of the information is already available in the utils package.

Reply:
It is to be passed by Auth Package to Utils, as these two Params are not Generic.
PS: Utils have all Generic Utility and Parameter values.